### ISSUE TITLE
Testing results on small validation set

### ISSUE THREAD

**Issue by Raspberry-beans** [SysCV/tet#6]
Hi,
Firstly thanks a lot for great work. Your model is basis for my current internship project!

I have done testing on validation split set containing 200 video sequences and I got the same results as mentioned in your paper as shown below:

![TETer_full](https://github.com/SysCV/tet/assets/69916197/f2124f23-6a65-43af-983f-00e7c55c149f)

I want to try testing on small validation split to save time. Hence I only chose first ten validation sequences, make the corresponding annotations using `python -m bdd100k.label.to_coco` command and run the testing. For the small validation split, the metrics like TETA, LoacA etc were heavily decreased. Even the MOTA is coming up in negative values (see figure below).

![TETer_small_edit](https://github.com/SysCV/tet/assets/69916197/415017b0-87dd-4a0e-bc3a-d37fb95908c5)

I am not sure about the reason for such low metric values for a smaller validation split. I only changed the `ann_file`, `scalabel_gt` and `img_prefix` paths to respective small validation split folder in `cem_bdd.py` configuration file. 

Can you guide me in this issue. Is this the expected output or I am missing something because of which my metrics are coming very low on small validation split. Any suggestion would be helpful for me. 

Best Regards,
Rehman

**Comment by siyliepfl** [comment]
Are you testing the model trained by yourself？

**Comment by Raspberry-beans** [comment]
No,
I am using `teter_bdd_r50_1x_20220706_134248.pth`, the same shared in the repo.

**Comment by siyliepfl** [comment]
In that case, if you only change the ann_file, scalabel_GT and get much lower results. I would recommend you to check your code for generating those small set annotations, especially for the ground truth file. Check if you have consistent ann_file and GT file, eg their class mapping , video name , image names…

**Comment by Raspberry-beans** [comment]
Thanks for your suggestions. I will look into the generated annotation file.

But I am not using a different code to generate annotations. I have used the same code that was used to generate annotations of actual validation split. 

I used the following command line mentioned in the repo.
`python  bdd100k.label.to_coco -m box_track -i bdd100k/labels/box_track_20/val_tiny -o data/bdd/annotations/box_track_20/box_track_val_tiny_cocofmt.json`

Only the input path is changed to tiny validation split `val_tiny` insted of actual validation set.
